md/hardware-and-software-requirements.md

```diff
@@ -17,57 +17,38 @@ Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
 ## Software
 
 Software required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Portal).
-| | 7.0.x <br/> 7.1.x | 7.2.x <br/> 7.3.x  | 7.4.x | 7.5.x
```

Comment:
Is this new presentation mode validated?
This is a drastic presentation change...